GUI Window which shows an MFD

YourAgony opened the pull request:
Hello,

I'm looking for a plugin which opens a window via Toolbar Button in which I have an RPM MFD and that i can use without having to go to the IVA ... is there such a thing?

Someone maybe an idea? or maybe someone contact to the Linuxgurugamer?

Thanks in advance ... YourAgony

Commit:
DeepFreeze Compatible integration - Wrapper class

Using DeepFreeze API wrapper class (no hard DLL dependency) this change
will make TAC LS aware of DeepFreeze mod and it's frozen kerbals. It
removes tracking of kerbals that are frozen so that LS resources are not
consumed when they are or when they are thawed using DeepFreeze mod.

james3838 committed:
KSP 1.1 fix

Update references
Change to new classes/methods

danfarnsy committed:
Merge pull request #3 from james3838/master

KSP 1.1 fix

danfarnsy committed:
Remove submodule prior to adding TacLib directly

danfarnsy committed:
Added TacLib directly instead of as submodule

danfarnsy committed:
Added cherrydev's fixes, KSP 1.1 compatible

danfarnsy committed:
Merge branch 'JPLRepo-master' into KSP11-update

danfarnsy committed:
Undid changes in previous commit

danfarnsy committed:
Folded in DFWrapper

danfarnsy committed:
Update versioning & license, use TMS textures

danfarnsy committed:
added search tags to parts

danfarnsy committed:
Included dependencies for release

danfarnsy committed:
Merge pull request #7 from KSP-RO/KSP11-update

Ksp11 update

danfarnsy committed:
Update TACLS version to 0.12.0

danfarnsy committed:
Update bundled dependencies, version number

Commit:
:lollipop: Added .gitattributes

Commit:
Initial Dev Commit split common KSP utilities to separate project

JPLRepo committed:
Merge pull request #1 from JPLRepo/Dev

Initial Dev Commit split common KSP utilities to separate project

Commit:
Update ModuleResourceConverter status to include new "Operational" status

danfarnsy committed:
update module manager

JPLRepo committed:
Merge pull request #2 from JPLRepo/Dev

Update ModuleResourceConverter status to include new "Operational" st…

danfarnsy committed:
Update CRP, MM, recompile for 1.1.3, and make texture refs consistent

Commit:
Fix DeepFreeze Integration

DeepFreeze integration broken.
https://github.com/JPLRepo/DeepFreeze/issues/58
TAC LS not recognising frozen kerbals as the wrapper integration was not
working.
Fixed.

JPLRepo committed:
Merge pull request #29 from JPLRepo/master

Fix DeepFreeze Integration

Commit:
Add SI - EC unit conversion routine.
Add new status for GenericConvert routine.
Add RO mod install routine.

Commit:
Merge branch 'master' into Dev

Commit:
Fix Log Spam https://github.com/KSP-RO/TacLifeSupport/issues/27

Commit:
Re-implement nuget pdb2mdb package. Cleanup of proj files for automation begun but not completed. Release processing needs work.
Added submodules REPOSoftTechKSPUtils in prep to add Common Library functions and Stock AppLauncher support.

Commit:
Fixed RO compatibility issue. Moved settings file out of PluginData - Write user settings into persistent save file. https://github.com/KSP-RO/TacLifeSupport/issues/5
Test deploy working, have to test release deploy on final release of next version .https://github.com/KSP-RO/TacLifeSupport/issues/6

Commit:
More Fixes to Deployment packages and MSBuild
Implemented Stock Applauncher support as well as Blizzy Toolbar support retained.
Re-Implemented Icon colour changes to the icons as well.

Commit:
Added Editor custom filter for TAC LS parts
Final clean-up

Commit:
Switch AVC ID back to the original now that we have control of it.

JPLRepo committed:
Merge pull request #35 from KSP-RO/Develop

V0.12.3.0

Commit:
Fix URL for submodule.

JPLRepo committed:
Merge pull request #36 from KSP-RO/Develop

Fix URL for submodule.

Commit:
v0.12.4
Fixed Settings file not being read correctly and using default values. (Typo - fat fingers)
 VAB/SPH Editor Category for TAC LS parts can be toggled on or off via the Settings menu (Hit the TAC LS icon in the Space Center).
 https://github.com/KSP-RO/TacLifeSupport/issues/37
 Fixed Rescue Kerbals when they EVA out (on rescue missions) to actually have Life Support resources. https://github.com/KSP-RO/TacLifeSupport/issues/37
 Fixed CKAN version issues.

JPLRepo committed:
Merge pull request #38 from KSP-RO/Develop

v0.12.4

Commit:
Support for different life support settings per save game, and also support MM override by Realism Overhaul.
So LifeSupport.Cfg file is now base settings for start of a NEW save only.
Once a save is started copies of these fields are persisted into the sfs save file.
Changes to existing saves must then only be made in-game and these changes are persisted into the sfs save file.

Commit:
Forgot the DLL

Commit:
Fixes for KSP 1.2

Commit:
Fix Resource request for KSP 1.2

Commit:
First commit KSP 1.2 fixes

JPLRepo committed:
Merge pull request #44 from JPLRepo/Develop

First commit KSP 1.2 fixes

Commit:
Merge branch 'Develop' of https://github.com/KSP-RO/TacLifeSupport into Develop
KSP 1.2 work continues...

Commit:
KSP 1.2 more request resource changes

Commit:
KSP 1.2 Develop

Commit:
KSP 1.2 develop continues...

Commit:
More KSP 1.2 work.

Commit:
KSP 1.2 Test Candidate

Commit:
Recompile

Commit:
Moved Textures to Icons folder (as that's what they are)
De-Linqing continues...
Fix Editor Filter?
Fix Install Checker for new folder locations and delete old TacLifeSupport.cfg file.
Move Monitoring Window vessel sorter out of onGUI (terrible performance there) and made it event driven.
Replaced MiniAVC with KSP 1.2 compatible version.
Removed redundant source files from TacLib.
Moved DLL from TACLifeSupport to TACLifeSupport/Plugins folder in project file.

Commit:
More fixes to Difficulty Settings. Integration to DeepFreeze and convert all resource converters to use stock ModuleResourceConverter.

JPLRepo committed:
Merge pull request #49 from KSP-RO/Develop

V0.12.5 for KSP 1.2

Commit:
v0.12.6 Candidate
Fix PreLaunch Vessel handling. Any vessel that is PreLaunch will display as such and will not consume resources (with the exception of ElectricCharge),
 however, TACLS will not track EC drain for a PreLaunch vessel so it is possible if you leave a vessel on the pad with no EC generation that immediately on launch
 TACLS will see the vessel as having no EC available. https://github.com/KSP-RO/TacLifeSupport/issues/50
 Fixed [UNKNOWN] vessel type showing for vessels in the UI.
 Fixed Vessel list not showing the active vessel at the top of the list when switching between vessels. https://github.com/KSP-RO/TacLifeSupport/issues/51
 Massive code clean-up for performance improvements begins. Internally TAC LS now uses KSP GameEvents for all vessel processing setting things up for future
 version resource processing of ALL vessels including unloaded vessels.
 Fix Tourist processing/monitoring. https://github.com/KSP-RO/TacLifeSupport/issues/52

Commit:
Fixed part search tags.
Fixed turing TAC LS on/Off in a save file.
Added support for Cuminity Category Kit
Rebalanced smaller Recycles to support 3 kerbals.
Fixed Rescue and tourist kerbals.

Commit:
Fix Rescue Kerbals processing and monitoring.
Fixed NullReferenceException on the Air Filter in the editor scene.
Added builheadprofiles to all part configs (for size sorting in editor).
Aligned part names to include size in their names.

Commit:
Create RSTKSPGameEvents

Commit:
Added RSTKSPGameEvents dll and removed use of reflection for better integration with DeepFreeze mod.
 Added Frozen field to internal dictionaries and DeepFreeze frozen kerbals now don't disappear from monitoring windows

Commit:
v0.12.6 ready for KSP 1.2.1

JPLRepo committed:
Merge pull request #55 from KSP-RO/Develop

v0.12.6 for KSP 1.2.1

Commit:
Remove tracking of previously tracked vessels if they become empty of Crew. This eliminates tracking and warning messages.
Added conversionRate field to TACGenericConverter.

Commit:
Removed tracking of vessels that did have crew but now don't. - no more empty vessels being tracked.
 Added conversionRate field to TACGenericConverter. This is an optional field and defaults to a value of 1. this is to support numerous requests to add this feature
 back into to this module as it was removed when the module was changed to stock in v0.12.5. If you specify this field in your converter config file the value must be
 greater than zero (or it sets it back to the default) and can be any decimal value. Whatever value you specify will be used as a multiplier against all INPUT, OUTPUT and
 REQUIREMENTS config nodes and their specified resources.

Commit:
Change all TACLS recyclers to use the reinstated conversionRate field for TACGenericConverter.

Commit:
Update CRP to 0.6.2

Commit:
Update Change log.

Commit:
Tweak to EVA pack resources when Kerbal EVAs from regular vessel vs a Recovery Contract Vessel.

Commit:
Remove need for RSTKSPGameEvents dll.
Fix CrewType Tracking

Commit:
Finally fixed Rescue kerbal processing.

Commit:
Update changelog.

JPLRepo committed:
Merge pull request #59 from KSP-RO/KSP_1.2.2

Ksp 1.2.2

Commit:
Updated Community Resource Pack to version 0.6.3.
Updated Community Category Kit to 1.2.0.
If DeepFreeze mod is also installed Vessel Info UI window will show Totals for both Frozen and Unfrozen kerbals.

Commit:
Fix font bold for Frozen Crew display.
Fix monitoring of frozen crew.

JPLRepo committed:
Merge pull request #61 from KSP-RO/Develop

Develop

Commit:
Round all part costs off.
Fix syntax 2.75m Sabatier

JPLRepo committed:
Update .gitmodules

JPLRepo committed:
Merge pull request #64 from KSP-RO/Develop

Develop

Commit:
Fix TACLS and CCK part category filters. There can be only one.

Commit:
Cater for Life support values being changed in settings to correctly apply/adjust EVA kerbals
Changed handling of rescue kerbals to support kerbals that are already on EVA (not in a part). Rescue kerbals will not not consume resources until their vessel is Owned by the player. This occurs when the player switches to them or they come within loaded range of the active vessel (2.2km).
Show an info string in flight view for the mini-settings as the player cannot change them in flight view and they are disabled.

Commit:
Remove tracking for lost vessel (through docking or destruction).

Commit:
Nerfed Specialist Bonuses for all converters.

Commit:
Added EC consumption for EVA kerbal lights.

Commit:
update changelog, CCK and CRP

Commit:
Increments versions for release. 0.12.8

JPLRepo committed:
Merge pull request #72 from KSP-RO/Develop

version 0.12.8

Commit:
Fix absolute values on resource maths.
null checks on stock toolbar button.

Commit:
Fix sub project.
Fix rescue kerbal contracts cancelled.

Commit:
Fix Contract Kerbal check to only check first name of Kerbal.
Fix project files

Commit:
Fix Rescue contracts... again... for edge case where contract only contains the Kerbals First Name.
Added feature to fill EVA Kerbal's suit (and empty waste resources) when they leave an External Command Seat (resources are moved to/from the vessel the command seat is a part of).
Fixed out of range exception when vessels are removed from TAC LS tracking.

JPLRepo committed:
Update CCK and CRP. Fix Resoource Converters to 100% without specialist

JPLRepo committed:
Merge pull request #75 from KSP-RO/Develop

v0.12.9

JPLRepo committed:
Fix Rescue kerbal for kerbals on the ground due to contracts system not being loaded in time.
	Change Water purifier Water output rate to match water usage rates so as to avoid positive water scenario.
	Fix issue with Low Electricity Warning at high warp rate.

JPLRepo committed:
Woops. Need to actually start a Coroutine

JPLRepo committed:
Merge pull request #81 from KSP-RO/Develop

v0.13.0

JPLRepo committed:
Stop settings from resetting - Maybe

JPLRepo committed:
Step 1 - Compile

JPLRepo committed:
Update CCK 2.0.1 - KSP 1.3

JPLRepo committed:
Finalize for KSP 1.3

JPLRepo committed:
Merge branch 'KSP_1.3' into Develop

JPLRepo committed:
Merge pull request #85 from KSP-RO/Develop

Release for KSP 1.3.0

JPLRepo committed:
Fix NRE for probe vessels.

JPLRepo committed:
Merge pull request #86 from KSP-RO/Develop

Fix NRE for probe vessels.

JPLRepo committed:
Add Russian Localization and tag fixes

JPLRepo committed:
Fix EC processing for unloaded vessels.
Expanded Unloaded vessel resource processing to Produce EC for vessels that have solar panels and generators (including Kopernicus Solar Panels).
Russian Localization texts included- Thanks go to forum user Tirathangil.

Fitiales committed:
Update localization es-es & es-mx.cfg

JPLRepo committed:
Merge pull request #93 from Fitiales/patch-1

Update localization es-es & es-mx.cfg

JPLRepo committed:
Handle overflow resource caching on Unloaded vessels

JPLRepo committed:
Finally fix EC usage at high warp
Fix so opening the window only occurs in an actual atmosphere.
Added Hibernate option for EC and O2 consumption. Re-wrote how hibernation works internally.

JPLRepo committed:
Fix killing kerbals and Static PRessure (use outside air calcs)

JPLRepo committed:
Update to latest CRP

JPLRepo committed:
This is a DEV/Test build.

Fix so opening the window only occurs in an actual atmosphere.
Window will be opened when EC or O2 runs out and in an atmosphere.
Added Hibernate option for EC and O2 consumption. Re-wrote how hibernation works internally.
Hibernation vs Death now working correctly.
Fix EC processing for unloaded vessels.
Rescue Kerbals now have correct Life support given to them. Rescue kerbas will remain as "Rescue Me!" status consuming no resources until such time as either a) - You swap to them or b) they are loaded (come within loading range of the active vessel).
Fixed issue where recovered vessels at times will not clean up Kerbal tracking causing multiple "copies" and duplicated kerbals appearing in the TAC LS monitoring UI.

JPLRepo committed:
Fix Difficulty Settings

JPLRepo committed:
Merge pull request #98 from KSP-RO/Develop

v0.13.5

JPLRepo committed:
KSP 1.3.1

JPLRepo committed:
Merge pull request #101 from KSP-RO/Develop

KSP 1.3.1

JPLRepo committed:
Fix issue where if CCK is not installed TAC LS parts not showing in Editors.
Fix issues with EC consumption - Correct delta time check and not recording last EC usage when EC runs out resulting in never ending shortage in both scenario where window can be open and cannot.

JPLRepo committed:
V 0.13.8 for KSP 1.4.1

JPLRepo committed:
Merge pull request #107 from KSP-RO/Develop

V 0.13.8 for KSP 1.4.1

JPLRepo committed:
Fix Life support for new EVA suits
Fix module manager version

JPLRepo committed:
Merge pull request #108 from KSP-RO/Develop

Fix Life support for new EVA suits

bruceCzK committed:
Add Chinese Translation

Partially

JPLRepo commented:
Incorrect. Why are you trying to merge an old repo to the new master... Completely incorrect.
for those that have access.
DO NOT MERGE THIS UNDER ANY CIRCUMSTANCES.

YourAgony commented:
Sorry, but I had a few technical problems with my PC, so there was a lot of confusion here

JPLRepo committed:
Merge pull request #109 from bruceCzK/master

Add Chinese Translation

JPLRepo committed:
Update for KSP 1.5.x

JPLRepo committed:
Merge pull request #115 from KSP-RO/Develop

Update for KSP 1.5.x

JPLRepo committed:
add MH parts to the crewable part tests

JPLRepo committed:
Fix female Kerbal EVA LS resources when no Making History DLC is installed.

JPLRepo committed:
Update version to 0.13.11

JPLRepo committed:
Merge pull request #116 from KSP-RO/Develop

Fixes for Making History

JPLRepo committed:
Update for KSP 1.6.x

JPLRepo committed:
update changelog for v0.13.12

JPLRepo committed:
Merge pull request #121 from KSP-RO/Develop

Develop to Master for KSP 1.6.x

Commit:
Added bulkheadProfiles to TacWasteContainer

4x4cheesecake committed:
Don't use hardcoded day and year lengths

Adds support for Kronometer and still works fine with stock settings for Kerbin/Earth clocks.

JPLRepo committed:
Merge pull request #122 from lampeh/bulkheadfix

Missing bulkheadProfiles in TacLifeSupportContainers/Waste.cfg

JPLRepo committed:
Merge pull request #124 from 4x4cheesecake/master

Kronometer support

JPLRepo committed:
remove log spam on scene change

JPLRepo committed:
Update versions for KSP 1.7.x

JPLRepo committed:
remove some more log spam

JPLRepo committed:
Update backgroundresources

JPLRepo committed:
Merge pull request #125 from KSP-RO/Develop

Develop

JPLRepo committed:
Updating for KSP 1.8.x

JPLRepo committed:
Add warning and critical warning percentages to settings.

JPLRepo committed:
Merge pull request #130 from KSP-RO/Develop

Develop to master

JPLRepo committed:
Update included mods

JPLRepo committed:
Updates for KSP 1.9.x

JPLRepo committed:
Merge pull request #133 from KSP-RO/Develop

Update for KSP 1.9.x
